### DISCUSSION TITLE
Fluent version 6.0.0.0 is slower then 2.1.0.0

### DISCUSSION THREAD

**Discussion by vikastarlekar** [fluentribbon/Fluent.Ribbon#915] [General]
Hi ,

I am using fluent.dll since 2012 .When I upgraded it to 6.0 version , I observed it is slower.
It there any known issue for performance in this version ?

**Comment by batzen** [comment]
The current version is 8. Do you experience slow performance there too?
What exactly is slow?

**Reply by vikastarlekar** [reply]
Hi, I am feeling slowness when I try to open any menu Item ,when I right click on any item it is taking time to load that menu item and also window is taking time to load which was not with version 2.1.0.0.
Actually I want colorful Title bars for all my ribbon widows to differentiate between environments and this feature is released in version 6.0  .

**Reply by batzen** [reply]
If you could provide some kind of repro or sample application where i could measure the performance, i could have a look at what got slower and why and hopefully improve the performance.

On my machine the menu and context menu is shown instantly without any noticeable delay.

Could you check if you also notice this in the Fluent.Ribbon showcase application?
